### PR TITLE
Updating images and directions for latest ApiPort release

### PR DESCRIPTION
**Pull request by conniey**
# Updating Portability Analyzer documentation

## Summary

Updating the portability analyzer images and instructions for our latest release.

## Suggested Reviewers
@mairaw

### CODE REVIEW

File: docs/standard/portability-analyzer.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ After running the analysis, you will see your .NET Portability Report. Only type
...
 
-*   Type the following command to analyze the current directory: `\...\ApiPort.exe .`
-*   To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe first.dll second.dll third.dll`
+*   Type the following command to analyze the current directory: `\...\ApiPort.exe analyze -f .`
+*   To analyze a specific list of .dll files, type the following command: `\...\ApiPort.exe analyze -f first.dll -f second.dll -f third.dll`
 
 Your .NET Portability Report will be saved as an Excel file (*.xlsx*) in your current directory. The **Details** tab in the Excel Workbook will contain more information.
 
-For more info on the .NET Portability Analyzer, see the article [Leveraging existing code across .NET platforms](https://blogs.msdn.microsoft.com/dotnet/2014/08/06/leveraging-existing-code-across-net-platforms/) on the .NET blog.
+For more info on the .NET Portability Analyzer, check out the [GitHub documentation](https://github.com/Microsoft/dotnet-apiport#documentation) and [A Brief Look at the .NET Portability Analyzer](https://channel9.msdn.com/Blogs/Seth-Juarez/A-Brief-Look-at-the-NET-Portability-Analyzer) Channel 9 video.
```
Comment (mairaw) [inline docs/standard/portability-analyzer.md]: info -> information
check out -> a bit too informal. Change to see or visit

**Review by mairaw — COMMENTED** [review]
The changes LGTM overall. Please change the ms.date to today's date on the metadata section.
Also, does this work with any VS version?

**Comment by conniey** [comment]
@mairaw I fixed the document based on your comments.  In our latest release of .NET Portability Analyzer VS Extension, it only supports VS 2015.

**Comment by mairaw** [comment]
@conniey apologies on the delay here. So should we mention that VS version somewhere in the form of Prerequisites or some note?

**Comment by conniey** [comment]
@mairaw I added 2015 to "The Portability Analyzer is offered as a Visual Studio 2015 Extension and as a console app." When they click the [link for the extension](http://go.microsoft.com/fwlink/?LinkID=507467), the marketplace website also notes the installation targets for the extension.

Is that sufficient?

**Comment by mairaw** [comment]
Yes, that works @conniey. Thanks! I'll wait for the CI to finish and merge this.

**Pull request merged**